src/new/wallet.js

```diff
@@ -558,105 +566,8 @@ class HathorWallet extends EventEmitter {
       this.emit('update-tx', newTx);
     }
     return;
-
-    // TODO XXX Uncomment the following block to keep track of the balance.
-    /*
-    // We need to reload it because it was modified by updateHistoryData.
-    const newWalletData = wallet.getWalletData();
-    const { keys } = newWalletData;
-
-    const updatedHistoryMap = {};
-    const updatedBalanceMap = {};
-    const balances = this.getTxBalance(newTx);
-
-    // we now loop through all tokens present in the new tx to get the new history and balance
-    for (const [tokenUid, tokenTxBalance] of Object.entries(balances)) {
-      // we may not have this token yet, so state.tokensHistory[tokenUid] would return undefined
-      const currentHistory = state.tokensHistory[tokenUid] || [];
-      const newTokenHistory = addTxToSortedList(tokenUid, tx, tokenTxBalance, currentHistory);
-      updatedHistoryMap[tokenUid] = newTokenHistory;
-      // totalBalance should not be confused with tokenTxBalance. The latter is the balance of the new
-      // tx, while the former is the total balance of the token, considering all tx history
-      const totalBalance = getBalance(tokenUid);
-      updatedBalanceMap[tokenUid] = totalBalance;
-    }
-    const newTokensHistory = Object.assign({}, state.tokensHistory, updatedHistoryMap);
-    const newTokensBalance = Object.assign({}, state.tokensBalance, updatedBalanceMap);
-
-    storage.setItem('local:tokens:history', newTokensHistory);
-    storage.setItem('local:tokens:balance', newTokensBalance);
-    */
   };
```

Comment:
Why was `sendMultiTokenTransaction` removed?

Reply:
This method was not used. This was actually created to be used in an event a long time ago and it does not work. Maybe I should've removed it in another PR but it also used PIN, that's why I decided to remove it instead of refactoring it.